api/controllers/InviteController.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * MedicalProfessionalController
+ *
+ * @description :: Server-side actions for handling incoming requests.
+ * @help        :: See https://sailsjs.com/docs/concepts/actions
+ */
+
+const { applicationExists, getApplication } = require('../utils/DataStore');
+
+module.exports = {
+  index: function (req, res) {
+    /**
+     * If there is a medical report in the session, load it
+     */
+    if (req.session.medicalReport) {
+      res.locals.data = req.session.medicalReport;
+    }
+
+    // Get code, then clear session
+    const applicationCode = req.session.medicalReport.applicationCode;
+    req.session.medicalReport = {};
+
+    // Get full URL
+    const protocol = req.connection.encrypted? 'https' : 'http';
+    const medicalProfessionalRoute = sails.config.routes['GET /en/medical-professional'].i18n[req.i18n.locale];
```

Comment:
you can use the route helper on the sails object here: `sails.route('medical-professional');`

Reply:
Good call! Will make this update on a follow-up PR later today and merge this version to make sure we get this in for this afternoon.